core/domain/exp_jobs_one_off.py

```diff
@@ -205,7 +209,8 @@ def map(item):
             else:
                 exploration.validate(strict=True)
         except utils.ValidationError as e:
-            yield (item.id, unicode(e).encode(encoding='utf-8'))
+            yield (item.id, python_utils.convert_to_bytes(e).encode(
```

Comment:
Encoding here is redundant, just yield the bytes.

Reply:
Done